modules/framework/galasa-parent/dev.galasa.framework.api.resources/src/main/java/dev/galasa/framework/api/resources/ResourcesServlet.java

```diff
@@ -71,7 +72,9 @@ public void init() throws ServletException {
 			CPSFacade cpsFacade = new CPSFacade(framework);
 			ICredentialsService credsService = framework.getCredentialsService();
 			RBACService rbacService = framework.getRBACService();
-            addRoute(new ResourcesRoute(getResponseBuilder(), cpsFacade, credsService, timeService, rbacService));
+			IConfigurationPropertyStoreService cpsService = framework.getConfigurationPropertyService("framework");
```

Comment:
We already have a CPS facade in this class which can perform CPS operations on the `framework` namespace - for streams, we could just use the streams service